src/app/public/modules/progress-indicator/progress-indicator-status-marker/progress-indicator-status-marker.component.ts

```diff
@@ -40,13 +36,13 @@ import {
 export class SkyProgressIndicatorStatusMarkerComponent implements OnDestroy {
 
   @Input()
-  public set displayMode(value: SkyProgressIndicatorDisplayMode) {
+  public set displayMode(value: 'vertical' | 'horizontal') {
```

Comment:
Why not type this argument with your new display mode type? If we make changes to `SkyProgressIndicatorDisplayModeType`, wouldn't this get out of sync?